fmriprep/cli/run.py

```diff
@@ -119,7 +119,9 @@ def get_parser():
              ' - T1w: subject anatomical volume\n'
              ' - template: normalization target specified by --template\n'
              ' - fsnative: individual subject surface\n'
-             ' - fsaverage*: FreeSurfer average meshes'
+             ' - fsaverage*: FreeSurfer average meshes\n'
+             'this argument can be single value or a space delimited list\n'
```

Comment:
This `\n` is not respected in Sphinx: https://3347-53175327-gh.circle-artifacts.com/0/home/circleci/out/docs/usage.html

Adding a comma, to make it work in that case as well as in the CLI help.